src/integration/impl/circleci/StubbedCircleCiBackbone.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Stubbed/placeholder CircleCI backbone
+ */
+class StubbedCircleCiBackbone {
+
+    /**
+     * Creates a new stubbed RSS backbone
```

Comment:
nit: CircleCI instead of RSS

Reply:
Whoops ps this line is wrong in the travisci file too I think as thats where I copied from ;)